include/motor-control/core/motor_control.hpp

```diff
@@ -0,0 +1,123 @@
+#pragma once
+
+#include "common/core/bit_utils.hpp"
+#include "common/firmware/motor.h"
+#include "spi.hpp"
+
+namespace motor_control {
+
+const uint8_t WRITE = 0x80;
+/**
+ * Implementation of the Motor concept.
+ *
+ */
+
+enum class MotorRegisters : uint8_t {
```

Comment:
to me the definitions belong in some other Controller type of class; it makes sense to keep the two concepts
- Motion controller, which takes values like velocity, distance, etc and turns them into pulse trains
- Motor driver controller, which sets values like chopper style, etc
separate, since they actually do pretty separate things.

Reply:
I think that change should be made in a follow up PR ^